templates/ep19/bs/talks/talk.html

```diff
@@ -20,24 +20,24 @@
 <div class="container page" id="talk_page">
     <nav aria-label="breadcrumb">
         <ol class="breadcrumb">
-            <li class='breadcrumb-item'><a href="/">Home</a></li>
-            <li class='breadcrumb-item'><a href='#'>Talks</a></li>
-            <li class='breadcrumb-item active'>{{ talk_as_dict.title }}<li>
+            <li class="breadcrumb-item"><a href="/">Home</a></li>
+            <li class="breadcrumb-item"><a href="/events/sessions/#Talks">Talks</a></li>
+            <li class="breadcrumb-item active">{{ talk_as_dict.title }}<li>
         </ol>
     </nav>
 
     <div class="row">
         <div class="col-md-12">
-            <div class='talk'>
+            <div class="talk">
                 <h1>{{ talk.title }}</h1>
                 <h3>{{ talk.sub_title }}</h3>
                 <h5>{% for speaker in talk_as_dict.speakers %}
-                    {{ speaker.name }}{% if not forloop.last %}, {% endif %}
+                    <a href="{{ speaker.url }}"></a>{{ speaker.name }}</a>{% if not forloop.last %}, {% endif %}
```

Comment:
This will require adding user profile url here, which iirc is not currently stored in the speaker.url; possible solutions: (1) add url to the dict dump in in conference/talks.py or (2) replace with `{% url %}` to the speaker page.

Reply:
On a separate note, the anchor tag is closed twice.

Reply:
Oh, what a shame! :worried: Smart IDEs and me. I should have used vim! :smile:

Reply:
I'll try to add url to the speaker object. That seems the cleanest to me.

Reply:
I hope that is now fixed. Didn't try out hands-on whether referencing the slug works that way. :grimacing: